Scripts/WikiUpdater.cs

```diff
@@ -0,0 +1,379 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using AngleSharp.Html.Dom;
+using Scripts;
+using ScriptsBase.Models;
+using ScriptsBase.Utilities;
+
+public static class WikiUpdater
+{
+    private const string ORGANELLE_CATEGORY = "https://wiki.revolutionarygamesstudio.com/wiki/Category:Organelles";
+    private const string WIKI_FILE = "simulation_parameters/common/wiki.json";
+    private const string ENGLISH_TRANSLATION_FILE = "locale/en.po";
+
+    /// <summary>
+    ///   Compounds to replace with custom BBCode when appearing in bold on wiki pages.
+    /// </summary>
+    private static readonly string[] CustomBbcodeCompounds =
+    {
+        "ATP",
+        "Ammonia",
+        "Carbon Dioxide",
+        "Glucose",
+        "Hydrogen Sulfide",
+        "Iron",
+        "Mucilage",
+        "Nitrogen",
+        "Oxygen",
+        "OxyToxy",
+        "Phosphates",
+        "Sunlight",
+        "Temperature",
+    };
+
+    /// <summary>
+    ///   Inserts selected content from the online wiki into the game files. See
+    ///   https://wiki.revolutionarygamesstudio.com/wiki/Thriveopedia for instructions.
+    /// </summary>
+    public static async Task<bool> Run(CancellationToken cancellationToken)
+    {
+        var organellesRootTask = FetchOrganellesRootPage(cancellationToken);
+        var organellesTask = FetchOrganellePages(cancellationToken);
+
+        var organellesRoot = await organellesRootTask;
+        var organelles = await organellesTask;
+
+        var untranslatedWiki = new Wiki(
+            organellesRoot.UntranslatedPage,
+            organelles.Select(o => o.UntranslatedPage).ToList());
+
+        await JsonWriteHelper.WriteJsonWithBom(WIKI_FILE, untranslatedWiki, cancellationToken);
+        ColourConsole.WriteSuccessLine($"Updated wiki at {WIKI_FILE}, running translations update");
+
+        var localizationUpdater = new LocalizationUpdate(new LocalizationOptionsBase { Quiet = true });
+        if (!localizationUpdater.Run(cancellationToken).Result)
```

Comment:
```suggestion
        if (!await localizationUpdater.Run(cancellationToken))
```
Shouldn't this be awaited like this? Or am I misremembering how the run method was defined?

Reply:
Well the code like it was before would block the current thread to just wait on that single async operation, rather than yielding the current thread back to the async thread pool, which is the correct thing to do.